.github/scripts/github_prepare_release.sh

```diff
@@ -5,12 +5,11 @@ _root_dir=$(dirname $(greadlink -f $0))
 _chromium_version=$(cat $_root_dir/ungoogled-chromium/chromium_version.txt)
 _ungoogled_revision=$(cat $_root_dir/ungoogled-chromium/revision.txt)
 _package_revision=$(cat $_root_dir/revision.txt)
-_epoch_finish=$(date +%s)
 
 _x64_file_name="ungoogled-chromium_${_chromium_version}-${_ungoogled_revision}.${_package_revision}_x86-64-macos.dmg"
 _arm64_file_name="ungoogled-chromium_${_chromium_version}-${_ungoogled_revision}.${_package_revision}_arm64-macos.dmg"
-_release_tag_version="${_chromium_version}-${_ungoogled_revision}.${_package_revision}__${_epoch_finish}"
-_release_name="Ungoogled-Chromium macOS ${_release_tag_version}"
+_release_tag_version="${_chromium_version}-${_ungoogled_revision}.${_package_revision}"
+_release_name="${_release_tag_version}"
```

Comment:
I think this variable can be removed now, since it's now just an alias for `_release_tag_version`.

Other changes necessary for this:
1. Remove line 17 `echo "release_name=$_release_name" >> $GITHUB_OUTPUT`
2. Replace `name: ${{ steps.bake.outputs.release_name }}` with `tag_name: ${{ steps.bake.outputs.release_tag_version }}` in .github/workflows/build.yml`

Reply:
I'd prefer to use the exact name for the outputs, even though they have the same content. Just a personal preference.